test/mock/mock_api.py

```
@@ -0,0 +1,94 @@
+
+
+def query_generator(conversation_id: int, usr_query: str):
+    import time
+    response = f'\n# Conversation **{conversation_id}**\nResponse for: *{usr_query}*'
+    for c in response:
+        time.sleep(0.1)
+        yield c
+
+
+@app.post('/conversations/{conversation_id}/chat')
+async def query(conversation_id: int, body: dict = Body(...)):
```

Ruff (0.8.2)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)